src/fetch/FetchExample.re

```diff
@@ -0,0 +1,93 @@
+type tDog = string;
+
+type tArrayDogs = array(tDog);
```

Comment:
Here too, while I suppose this saves 1 character, it otherwise seems to serve no purpose, but on the other hand obfuscate the actual type. When I come across `tArrayDogs` elsewhere, I now have to look it up to ensure it has the type I think it's supposed to have. The alias actually makes me more unsure, because I don't understand what purpose it serves.